packages/quantic/cypress/integration/refine-toggle/refine-toggle.cypress.ts

```diff
@@ -67,6 +89,40 @@ describe('quantic-refine-toggle', () => {
         Expect.displayModal(false);
       });
     });
+
+    describe('when no facet is registered', () => {
```

Comment:
Given ln195 defines a `describe` block with the same title, I assume this one should be:

```suggestion
    describe('when facets are registered', () => {
```